dbt/adapters/sqlserver/connections.py

```diff
@@ -167,8 +170,13 @@ def open(cls, connection):
                 con_str.append(f"UID={{{credentials.UID}}}")
                 con_str.append(f"PWD={{{credentials.PWD}}}")
 
-            if not getattr(credentials, "encrypt", False):
-                con_str.append(f"Encrypt={credentials.encrypt}")
+            # still confused whether to use "Yes", "yes", "True", or "true"
+            # to learn more visit
+            # https://docs.microsoft.com/en-us/sql/relational-databases/native-client/features/using-encryption-without-validation?view=sql-server-ver15
+            if getattr(credentials, "encrypt", False) is True:
+                con_str.append(f"Encrypt=Yes")
+                if getattr(credentials, "trust_cert", False) is True:
+                    con_str.append(f"TrustServerCertificate=Yes")
```

Comment:
@mikaelene, I think @NandanHegde15 and I figured it out.
`not getattr(credentials, "encrypt", False)` currently evaluates to `False` no matter what string was passed to the `encrypt` param in the `profiles.yml`. Because any string evaluates to `True`.

Our fix was to make `encrypt` and `trust_cert` both boolean values. it should now work for you if you don't specify these in your profile.
```python
#this prints
if "no!":
 print("foobar")
```